Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -49,6 +49,7 @@ public IEnumerable<ContentItemTemplate> Templates
         }
 
         public bool LaunchDebugger { get; set; }
+        public bool MultithreadMode { get; set; }
```

Comment:
I see that you were copying the way `LaunchDebugger` was done, but the `Settings` class was added after this variable, and it would make more sense to just use the `Settings.Instance.variable` directly instead of adding an extra variable in the Controller.